test/openAPI/features/wallet_binding.feature

```diff
@@ -0,0 +1,21 @@
+@method=POST @endpoint=/wallet-binding
+Feature: The endpoint to validate wallet create wallet user id
+
+  @smoke @unit @positive
+  Scenario: Successfully validates the wallet and generates the wallet user id smoke type test
+    Given Wants to validate the wallet and generate wallet user id
+    When Send POST /wallet-binding request with given "n3fy2qkg9r7h2" as individualId and "OTP" as authFactorType and "encoded-json" as format and "string" as challenge
```

Comment:
publicKey is also required and cannot be empty. [Here](https://github.com/GovStackWorkingGroup/bb-identity/pull/27#discussion_r1197999292) is an example of publicKey